Gemfile

```diff
@@ -42,6 +42,10 @@ group :bench do
 end
 
 group :test do
+  # allows for rspec-like invocation of single tests
+  # e.g.: $ m test/path.rb:lineNum
+  gem 'm', '~> 1.4.2'
```

Comment:
lulz, I have that in my `Gemfile.local`, which is why I added that part to the Gemfile. also where I configure pry and byebug